INSTALL.make.md

```diff
@@ -0,0 +1,257 @@
+QUICK INSTALLATION PROCEDURE.
+=============================
```

Comment:
Since this is a new file, please change the casing, and use the appropriate level for titles (which I think is `<h2>` rather than `<h1>` and thus should be done with `------------` or `##`.